code/modules/projectiles/guns/energy/special.dm

```diff
@@ -20,7 +20,7 @@
 	name = "ion carbine"
 	desc = "The MK.II Prototype Ion Projector is a lightweight carbine version of the larger ion rifle, built to be ergonomic and efficient."
 	icon_state = "ioncarbine"
-	w_class = WEIGHT_CLASS_NORMAL
+	w_class = WEIGHT_CLASS_BULKY
```

Comment:
I have to question what the point even is to this weapon if the ion rifle and ion carbine are the same size. I thought the weapon being baggable was it's main draw.

Reply:
You get the ion rifle shiftstart, whereas one carbines are researched you can print as many as you want.

Reply:
The carbine existed explicitly to be a smaller version of the standard rifle. There is no point to its existence now (besides being put on your belt instead of your back) that weapon sizes are universally nerfed. You are better off removing this one and pointing the research datum to the rifle instead.

Reply:
I think putting it on your belt/back is a pretty good reason to keep it separate. It's also good to have the unique one in the armory to know if the reserve's been taken.